README.md

```diff
@@ -44,6 +44,17 @@ You have two ways of telling you ROS 2 application which XML to use:
 1. Placing your XML file in the running directory under the name `DEFAULT_FASTRTPS_PROFILES.xml`.
 2. Setting environment variable `FASTRTPS_DEFAULT_PROFILES_FILE` to your XML file.
 
+### Change publication mode
+
+Another way to change easily the publication mode is to use the environment variable `ROS_PUBLICATION_MODE`.
```

Comment:
@JLBuenoLopez-eProsima I'm somewhat inclined to stick to `RMW` for a prefix: 
```suggestion
Another way to change easily the publication mode is to use the environment variable `RMW_PUBLICATION_MODE`.
```

Reply:
For the moment, I have changed the name to the one proposed by @ivanpauno. If later other rmw implementations start to use a similar variable to configure the publication mode we can consider updating this code.